book/src/design/gadgets/sha256/table16.md

```diff
@@ -192,9 +192,9 @@ For each block $M \in \{0,1\}^{512}$ of the padded message, $64$ words of $32$ b
 are constructed as follows:
 - The first $16$ are obtained by splitting $M$ into $32$-bit blocks $$M = W_0 || W_1 || \cdots || W_{14} || W_{15};$$
 - The remaining $48$ words are constructed using the formula:
-$$W_i = \sigma_1(W_{i-2}) \boxplus W_{i-7} \boxplus \sigma_0(W_{i-15}) \boxplus W_{i-16},$$ for $i = 17, \ldots, 64$.
+$$W_i = \sigma_1(W_{i-2}) \boxplus W_{i-7} \boxplus \sigma_0(W_{i-15}) \boxplus W_{i-16},$$ for $i = 16, \ldots, 63$.
 
-> Note: $0$-based numbering is used for the $W$ word indices.
+> Note: $0$-based numbering is used for the $W$ word indices; this differs from the NIST spec.
```

Comment:
The NIST spec uses 1-indexing for the message blocks (`i`), but 0-indexing for the words within each block and for the message schedule (`t`). From section 6.2.2:

> For i = 1 to N:
> 1. Prepare the message schedule, {W_t}:
>     W_t = M^i_t for 0 <= t <= 15

Reply:
Damn and blast, NIST!